thats_the_way.py

```diff
@@ -0,0 +1,17 @@
+from os import listdir
+from os.path import isfile, join
+
+
+def that_the_way(path):
```

Comment:
the name of the function should indicate what it does, you can specify the name of the exercise in a comment if you want to